omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,61 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',function ($scope,TagService,ngDialog) {
+    $scope.init = function(personUuid) {
+            $scope.thisPersonUuid = personUuid;
+          $scope.loadTags =function () {
+              var payload = {
+                  objectType: 'org.openmrs.Person',
+                  objectUuid: personUuid
+              };
+              TagService.getTags(payload).then(function (res) {
+                  $scope.tags = res;
+              });
+          }
+         $scope.loadTags();
```

Comment:
Note that loadTags() should not be a scope bound function, it's a member of the main enclosing function